src/hakunapi-jsonfg/src/test/java/fi/nls/hakunapi/geojson/hakuna/JSONFGFeatureCollectionWriterTest.java

```diff
@@ -0,0 +1,56 @@
+package fi.nls.hakunapi.geojson.hakuna;
```

Comment:
Package doesn't match that of `fi.nls.hakunapi.jsonfg.JSONFGFeatureCollectionWriter` which isn't conventional. Obviously package private/protected methods aren't visible to the unit test class but those don't seem to get tested.

Reply:
fixed - forgot to rename test packages